modules/UpgradeWizard/silentUpgrade_step1.php

```diff
@@ -846,7 +846,26 @@ function sugar_cached($file)
                 } else {
                     $new_upgrade->description .= ' Silent Upgrade was used to upgrade the instance.';
                 }
-                $new_upgrade->save();
+
+                // Running db insert query as bean save will throw logic hook errors due to dependencies that are not set yet
+                $customID= create_guid();
```

Comment:
Please put a space after $customID
```suggestion
                $customID = create_guid();
```